src/applications/claims-status/components/ClosedClaimMessage.jsx

```diff
@@ -46,14 +47,14 @@ const getRecentlyClosedClaims = claims => {
       const { closeDate, open, phaseChangeDate } = claim.attributes;
 
       const isClosed = isEVSSClaim(claim) ? !open : Boolean(closeDate);
-      const closedDate = isEVSSClaim(claim) ? phaseChangeDate : closeDate;
+      const dateClosed = isEVSSClaim(claim) ? phaseChangeDate : closeDate;
       // END lighthouse_migration
 
       // If the claim is not an appeal, we want to filter it out
       // if it was closed more than 30 days ago
       return (
         isClosed &&
-        moment(closedDate)
+        moment(dateClosed)
```

Comment:
if `dateClosed` is `undefined`, moment will return today's date. Not sure if you need to check this or not... I've used something like `moment(dateClosed || null)` which then returns an invalid date.

Reply:
Good idea, new commit incoming